.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+# Default
+* @cdata
+* @ucan-wg/fission
```

Comment:
Just need @walkah or @cdata to give the team write access.

Reply:
fission team should have write access here now!

Reply:
validated.